### PR TITLE
Fix for FHIR-46282: Coverage self pay refactor

### PR DESCRIPTION
**Pull request by mjosborne1**
Fixes [FHIR-46282](https://jira.hl7.org/browse/FHIR-46282):
- Removed local CS and VS extensions on AU coverage.type bound valueset
- Added STU-Note about the VS expansions 
- Just noting the expansion of coverage-type does not contain the payconc code recently added to THO V 6.0.2

### CODE REVIEW

File: input/pagecontent/terminology.md [hunk 1]
```diff
@@ -3,6 +3,11 @@ This page lists all ValueSets and CodeSystems defined as part of this specificat
 
 As well as these included definitions this specification typically references [National Clinical Terminology Service (NCTS)](https://www.healthterminologies.gov.au/integration/R4/fhir) terminology resources. 
 
+
+<div class="stu-note">
+Expansions of ValueSets in this Implementation Guide may differ from expansions returned from interactions with a terminology server using the latest version of terminology.hl7.org (THO).
```
Comment (dt-r) [inline input/pagecontent/terminology.md:8]: Tweak suggested to the wording. "Implementers are advised to take note that expansions of value sets visible in this guide may differ from expansions returned with a server using [HL7 Terminology (THO)](http://terminology.hl7.org/6.0.2) version 6.0.0 or higher."

**Review by dt-r — CHANGES_REQUESTED** [review]
See inline comments.

File: input/pagecontent/terminology.md (hunk 1 above)
Comment (mjosborne1) [reply input/pagecontent/terminology.md]: Updated wording as per suggestion.

File: input/resources/au-coverage.xml [hunk 2]
```diff
@@ -32,8 +32,9 @@
       <path value="Coverage.type"/>
```
[the remark this answers is not in the document]
Comment (mjosborne1) [reply input/resources/au-coverage.xml]: Doh...Just need to save au-coverage.xml locally to fix this.

**Review by dt-r — APPROVED** [review]

**Pull request merged**